docs/plain-bundle-spec.md

```diff
@@ -0,0 +1,138 @@
+combines the type of bundle (plain) with the current schema version (v0).
+The [plain bundle provisioner](/internal/provisioner/plain/README.md) is able to source `plain+v0` bundles and install
+them onto a Kubernetes cluster.
+
+> Note: the plain+v0 bundle format is at schema version v0, which means it's an experimental format that is subject
+> to change.
+
+Supported source types for a plain bundle currently include the following:
+
+* A container image
+* A directory in a git repository
+
+Additional source types, such as a local volume or a generic URI-based resource, are on the roadmap. These source types
+all present the same content, a directory containing static Kubernetes YAML manifests, in a different ways.
+
+## Example
+
+For example, below is a minimal example of a Dockerfile that builds a `plain+v0` image-based bundle from a directory
+containing static Kubernetes manifests.
+
+```dockerfile
+FROM scratch
+COPY /manifests /manifests
+```
+
+where the given `manifests` directory contains the Kubernetes resources required to deploy an application, for example:
+
+```
+manifests
+├── namespace.yaml
+├── cluster_role.yaml
+├── role.yaml
+├── serviceaccount.yaml
+├── cluster_role_binding.yaml
+├── role_binding.yaml
+└── deployment.yaml
+```
+
+For an example of a directory backing a git-based bundle,
+see [the combo manifests directory](https://github.com/operator-framework/combo/tree/main/manifests). Any directory that
+contains only static Kubernetes manifests checked into a git repository accessible via a remote URL can be considered a
+plain bundle and sourced by the plain provisioner.
+
+> Note: there must be at least one resource in the manifests directory in order for the bundle to be a valid
+> plain+v0 bundle.
+
+## Technical Details
+
+* The static manifests must be located in the root-level /manifests directory in a bundle image for the bundle to be a
+  valid `plain+v0` bundle that the provisioner can unpack. A plain bundle image without a /manifests directory is
+  invalid and will not be successfully unpacked onto the cluster. For a git-based plain bundle, this limitation does not
+  exist, and the manifests can be in any directory in the repository. The manifest directory is assumed to be
+  ./manifests in a git-based plain bundle but can be overwritten.
+* The /manifests directory may contain additional sub-directories which contain static manifests.
+* The plain image-based bundle can be built from any base image, but `scratch` is recommended as it keeps the resulting
+  bundle image a minimal size.
+* Additional bundle metadata can be provided in two ways: either via image labels or by including a `/metadata`
+  directory inside the bundle. For a `plain+v0` bundle either is acceptable and entirely optional.
+* Including any content in the plain bundle that is not static YAML manifests will result in a failure when creating
+  content on-cluster from that bundle via a [BundleInstance](/README.md#BundleInstance).
```

Comment:
We should try to be as specific as we can here. For example:
- Are JSON manifests allowed?
- Are multiple objects allowed in a single file?
- Are specific file extensions expected?

This doc should include instructions (maybe TODO for now) for how someone can build and/or validate a plain+v0 bundle.